src/cli.js

```diff
@@ -1,14 +1,64 @@
 #!/usr/bin/env node
+let chalk = require('chalk')
+let hydrate = require('@architect/hydrate')
+let http = require('./http')
+let path = require('path')
 let sandbox = require('./index')
 let ver = require('../package.json').version
 let version = `Sandbox ${ver}`
+let watch = require('node-watch')
 let options = process.argv
-sandbox.start({version, options}, function _done(err) {
-  if (err && err.message === 'hydration_error') {
-    process.exit(1)
-  }
-  if (err) {
-    console.log(err)
-    process.exit(1)
-  }
-})
+
+start()
+function start() {
+  sandbox.start({version, options}, function watching(err) {
```

Comment:
I think we want to make sure that if we're restarting the sandbox, that we're closing everything down properly. [`sandbox.start` provides a second argument to the callback which is a function to 'close everything down'](https://github.com/architect/sandbox/blob/master/src/start.js#L121) - I think we want to use that in the case of restarting.

Reply:
This was intentional (for now); depending on your project size, shutting everything down can take quite a while. (Begin, for example, has a lot of tables, and can take a number of seconds to start up.) I'd like to move towards diffing the project upon change and restarting the specific service that needs to be restarted (as opposed to just http, or everything).